core-services/prow/02_config/_plugins.yaml

```
@@ -1010,6 +1010,12 @@ config_updater:
         - ci
       gzip: true
       name: ci-operator-misc-configs
+    ci-operator/config/stackrox/scanner/*-release-2.*.yaml:
+      clusters:
+        app.ci:
+        - ci
+      gzip: true
+      name: ci-operator-2.x-configs
```

I _think_ this is what you want: (ci-operator-2.x-configs would be a new configmap which does not exist)
```suggestion
      name: ci-operator-misc-configs
```